messaging/src/main/java/org/axonframework/eventhandling/pooled/MergeTask.java

```diff
@@ -115,6 +123,10 @@ protected CompletableFuture<Boolean> task() {
     }
 
     private CompletableFuture<TrackingToken> tokenFor(int segmentId) {
+        // Block the segment from being claimed by the Coordinator while we perform the merge.
+        // This prevents a race condition where the Coordinator might claim a segment that's being merged.
+        releasesDeadlines.put(segmentId, clock.instant().plusSeconds(60)); // Block for 1 minute (will be cleared after merge)
```

Comment:
Just a question: is it possible for the merge task to take longer than 60 seconds? I thought it's even faster. Why is it this arbitrary value?